### PR TITLE
Add 2.2.0.0 release note

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Add 2.2.0.0 release note

### Issues Resolved
https://github.com/opensearch-project/dashboards-maps/issues/17

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Shivamdhar — DISMISSED** [review]
Looks good, we also need a 2.2 branch in the repo. And this should be pushed to 2.2 as well : )

**Comment by junqiu-lei** [comment]
@Shivamdhar Thanks, let me create 2.2 branch.

File: release-notes/opensearch-dashboards-maps.release-notes-2.2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Version 2.2.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.2.0
+
+New features consist of two plugins, [geospatial](https://github.com/opensearch-project/geospatial) backend plugin for OpenSearch, and [dashboards-maps](https://github.com/opensearch-project/dashboards-maps) frontend plugin for OpenSearch Dashboards.
```
Comment (vamshin) [inline release-notes/opensearch-dashboards-maps.release-notes-2.2.0.0.md]: I think we can remove this. We don't have to mention about Geospatial plugin

**Review by vamshin — APPROVED** [review]
LGTM! Thanks

**Review by Shivamdhar — APPROVED** [review]

**Pull request merged**